src/internal/observable/fromEvent.ts

```diff
@@ -204,8 +204,8 @@ export function fromEvent<T, R>(
  * ```ts
  * import { fromEvent } from 'rxjs';
  *
- * const clicksInDocument = fromEvent(document, 'click', true); // note optional configuration parameter
- *                                                              // which will be passed to addEventListener
+ * // note optional configuration parameter which will be passed to addEventListener
+ * const clicksInDocument = fromEvent(document, 'click', { capture: true });
  * const clicksInDiv = fromEvent(someDivInDocument, 'click');
```

Comment:
This line fails as well. Can you please create a `div` so it doesn't fail? There are many examples in code base, e.g.

```ts
const div = document.createElement('div');
div.style.cssText = 'width: 200px; height: 200px; background: #09c;';
document.body.appendChild(div);
```

```suggestion
 * const clicksInDiv = fromEvent(div, 'click');
```

Reply:
@jakovljevic-mladen something like this?

```diff
import { fromEvent } from 'rxjs';
+
+ const div = document.createElement('div');
+ div.style.cssText = 'width: 200px; height: 200px; background: #09c;';
+ document.body.appendChild(div);

// note optional configuration parameter which will be passed to addEventListener
const clicksInDocument = fromEvent(document, 'click', { capture: true });
- const clicksInDiv = fromEvent(someDivInDocument, 'click');
+ const clicksInDiv = fromEvent(div, 'click');

clicksInDocument.subscribe(() => console.log('document'));
clicksInDiv.subscribe(() => console.log('div'));
```

Reply:
Yes.

Reply:
Done.